saifa/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  backend:
+    image: patranakamo/saifa-be:v1.3.3
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "9988:80"
+    volumes:
+      - ${APP_DATA_DIR}/data/app:/data
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/var/www/html/lnd:ro
+    depends_on:
+      - redis
+    environment:
+      APP_PASSWORD: $APP_PASSWORD
+      BITCOIN_NETWORK: $APP_BITCOIN_NETWORK
+      LN_SERVER_URL: "https://$LND_IP:$LND_REST_PORT"
+      REDIS_HOST: redis
+
+  frontend:
+    image: patranakamo/saifa-fe:v1.3.2
+    ports:
+      - "9989:80"
```

Comment:
The entire `ports` section can be removed. Instead you can add the app_proxy to this compose file as explained here: <add-link> and point it to port 80 of the frontend container.